src/codelabs/implicit-animations.md

```diff
@@ -207,20 +207,20 @@ example and click the **Show details** button to trigger the animation.
 The [Fade-in text effect][] example demonstrates the following features
 of `AnimatedOpacity`:
 
-* `AnimatedOpacity` listens for state changes in its `opacity` property.
-* Whenever `opacity` changes, `AnimatedOpacity` automatically animates the
+- `AnimatedOpacity` listens for state changes in its `opacity` property.
+- Whenever `opacity` changes, `AnimatedOpacity` automatically animates the
   widget's transition to the new value for `opacity`.
-* `AnimatedOpacity` requires a `duration` parameter to define the time it takes
+- `AnimatedOpacity` requires a `duration` parameter to define the time it takes
   to animate the transition between an old `opacity` value and a new one.
 
 {{site.alert.secondary}}
-  Note that implicit animations can only animate properties of a parent
-  `StatefulWidget`, so this example begins with the `FadeInDemo` widget that
+  Implicit animations can only animate properties of a parent
+  `StatefulWidget`. This example begins with the `FadeInDemo` widget that
   extends `StatefulWidget`.
 
-  Notice also that `AnimatedOpacity` animates a single property: `opacity`.
-  Some implicitly animated widgets can animate many properties, as the following
-  example illustrates.
+  The `AnimatedOpacity` animates a single property: `opacity`.
```

Comment:
Without the "Notice also" this reads a bit weird, as I'd expect it to be related to the other text in the same aside, but it's a separate point.

Reply:
**reply:** The whole thing is in a "note", so "Notice..." is redundant. I have a fix.